squad/core/management/commands/update_project_statuses.py

```diff
@@ -32,19 +32,36 @@ def add_arguments(self, parser):
             type=valid_date,
             help="Start date for project status updates (default: 6 months before current date, format: YYYY-MM-DD)."
         )
+        parser.add_argument(
+            '--date-end',
+            dest="date_end",
+            default=datetime.now(),
+            type=valid_date,
+            help="End date for project status updates (default: today, format: YYYY-MM-DD)."
+        )
 
     def handle(self, *args, **options):
         self.options = options
 
+        build_count = Build.objects.filter(
+            datetime__range=(timezone.make_aware(self.options['date_start']),
+                             timezone.make_aware(self.options['date_end'])),
+            status__finished=True
+        ).count()
+
+        logger.info("Updating ProjectStatus objects from builds...")
+        logger.info("Total build count in selected date range: %s" % build_count)
         builds = Build.objects.filter(
-            datetime__gt=timezone.make_aware(self.options['date_start']),
+            datetime__range=(
+                timezone.make_aware(self.options['date_start']),
+                timezone.make_aware(self.options['date_end'])),
             status__finished=True
         )
         total = builds.count()
-        for index, build in enumerate(builds):
+        for index, build in enumerate(builds.iterator()):
```

Comment:
index will always be 1, so there won't be any progress bar. This is what I'm getting:
```
[2019-02-18 11:28:01 +0000] [INFO] Updating ProjectStatus objects from builds...
[2019-02-18 11:28:01 +0000] [INFO] Total build count in selected date range: 99
[2019-02-18 11:28:01 +0000] [INFO] Progress:  1%[          ]
[2019-02-18 11:30:01 +0000] [INFO] Progress: 100%[##########]
```

Reply:
Nah it's because we update status every 100 records and you only have 99 in selected date range. Please try with more and if this still persists please let me know. We have to set the limit to something after all...

Reply:
:)